src/OOVPADatabase/D3D8/3911.inl

```diff
@@ -2449,21 +2449,28 @@ OOVPA_NO_XREF(D3DDevice_SetTransform, 3911, 14)
 // ******************************************************************
 // * D3D::UpdateProjectionViewportTransform
 // ******************************************************************
-OOVPA_XREF(D3D_UpdateProjectionViewportTransform, 3911, 1 + 9,
+OOVPA_XREF(D3D_UpdateProjectionViewportTransform, 3911, 1 + 11,
 
            XREF_D3D_UpdateProjectionViewportTransform,
            XRefOne)
 {
-    // mov esi, XREF_D3DDEVICE
+    // mov e??, XREF_D3DDEVICE
     XREF_ENTRY(0x0C, XREF_D3DDEVICE), // Derived
 
-    // push ebp
-    // mov  ebp, esp
-    OV_MATCH(0x00, 0x55, 0x8B, 0xEC),
-
     // and  esp, 0FFFFFFF0h
     // sub  esp, 58h
     OV_MATCH(0x03, 0x83, 0xE4, 0xF0, 0x83, 0xEC, 0x58),
+
+    // Start of assembly code group since anything afterward does change over time.
+    // mov e??,
+    OV_MATCH(0x0A, 0x8B),
+    // mov eax, [e?? + 0x?]
+    OV_MATCH(0x10, 0x8B),
+    // fild [ e?? + 0x?]
+    OV_MATCH(0x16, 0xDB),
```

Comment:
Add new line below this since there's 2 instructions between flid and jge.

Reply:
done